joomla/joomla-cms
# com_search on back-end: Results links opens in backend window instead of new Tab

Pull Request for Issue #11852  .

### Summary of Changes
In the list of search results the link to the search now opens in a new window
An icon is added to visually indicate that it is opening in a new window and a screen reader only text link is also present.

### Testing Instructions
Enable search results in the admin search component
Perform a few searches in the front end
Return to the search component in the admin and you will see the log of the searched terms.
The number at the end of the row is the number of times that the term has been searched for.
Click on the link and it will open in a new window/tab
